keyboards/crkbd/keymaps/frogpad/keymap.c

```diff
@@ -0,0 +1,176 @@
+#include QMK_KEYBOARD_H
+#include "g/keymap_combo.h"
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+#define _FROG 0
+#define _SYM 1
+#define _NUM 2
+#define _EXT 3
```

Comment:
we heavily recommend enums instead of defines for layers.
```suggestion
enum custom_layers {
    _FROG,
    _SYM,
    _NUM,
    _EXT,
};
```